src/protocol/xn297dump_nrf24l01.c

```diff
@@ -0,0 +1,294 @@
+/*
+ This project is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Deviation is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Deviation.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "common.h"
+#include "interface.h"
+#include "config/model.h"
+#include "rftools.h"
+
+#ifdef PROTO_HAS_NRF24L01
+#if SUPPORT_XN297DUMP
+
+#ifdef MODULAR
+    struct Xn297dump xn297dump;
+#endif
+
+#define INITIAL_WAIT       500
+#define ADDRESS_LENGTH     5
+#define PERIOD_DUMP        100
+#define MAX_PACKET_LEN     32
+#define CRC_LENGTH         2
+#define MAX_RF_CHANNEL     84
+#define DUMP_RETRIES       10  // stay on channels long enough to capture packets
+#define INTERVAL_AVERAGE   4
+
+static const char *const xn297dump_opts[] = {
+    _tr_noop("Bitrate"), "1 Mbps", "2 Mbps", "250 Kbps", NULL,
+    _tr_noop("Address"), "5 byte", "4 byte", "3 byte", NULL,
+    _tr_noop("Scrambled"), _tr_noop("No"), _tr_noop("Yes"), NULL,
```

Comment:
I'd set Scrambled to Yes by default as this is the most used mode

Reply:
That's what I thought too, but XN297_SCRAMBLED is 1 in iface_nrf24l01.h, and I didn't like the idea of defining it reversed within the protocol. If I rename the protopt "Unscrambled it might be  even more confusing. Suggestion?

Reply:
My two cents is that the values don't need to match.  The default option Yes is a presentation "thing" that means the code does whatever it needs to enable scrambling.

Reply:
No big deal, my changes for xn297@250kbps with cc2500 will require to explicitly state scrambling, I'll swap the on/off value if required then.
https://github.com/DeviationTX/deviation/pull/860/commits/78e85a24e18a9376d2f819e85c2c1c6e8cb7fdc8